src/devices/PiJuice/Models/LEDBlink.cs

```diff
@@ -29,21 +29,21 @@ public class LedBlink
         /// <summary>
         /// Color for first period of blink
         /// </summary>
-        public Color RGB1 { get; set; }
+        public Color ColorFirstPeriod { get; set; }
 
         /// <summary>
         /// Duration of first blink period in milliseconds between [10 – 2550]
         /// </summary>
-        public int Period1 { get; set; }
+        public int FirstPeriod { get; set; }
```

Comment:
Didn't catch this one before but if you have timing, on public classes, then please use TimeSpan.  Especially here with a range than is more than few milliseconds. And great that you put the range. You will have to adjust a bit the comment as well.